### PR TITLE
Tanka can't handle top level arrays 

### PR DESCRIPTION
**Pull request by GabrielDyck**
Hi @sh0rez.
I'm glad to help with this issue https://github.com/grafana/tanka/issues/310. Please, let me know if I forgot something or we need to add anything else.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 dist
+.idea
```
Comment (sh0rez) [inline .gitignore:2]: This is editor specific, please consider adding it to your private global gitignore: https://blog.sebastian-daschner.com/entries/global-gitignore instead :)

File: pkg/tanka/parse.go [hunk 2]
```diff
@@ -140,7 +140,7 @@ func evalJsonnet(baseDir string, env *v1alpha1.Config, extCode map[string]string
 		return nil, err
 	}
 
-	var dict map[string]interface{}
+	var dict interface{}
```
Comment (sh0rez) [inline pkg/tanka/parse.go:143]:
```suggestion
	var data interface{}
```

Given it's not a dict anymore, let's call it data :)

File: pkg/tanka/parse_test.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package tanka
+
+import (
+	"github.com/grafana/tanka/pkg/spec/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestEvalJsonnet(t *testing.T) {
```
Comment (sh0rez) [inline pkg/tanka/parse_test.go:9]: Would love to see this also check objects, and evaluation results.

You could for example create multiple pseudo "environments" in testdata, like so:

```
- /
- /jsonnetfile.json
- /cases/
- /cases/array/main.jsonnet
- /cases/object/main.jsonnet
```

Then evaluate these as a table test, checking the error is nil and the result is what we expect.
Good work!

Comment (GabrielDyck) [reply pkg/tanka/parse_test.go:9]: Great! I'll add it. Thank you for your feedback.

File: .gitignore (hunk 1 above)
Comment (GabrielDyck) [reply .gitignore]: Great advice. Thanks!

**Comment by GabrielDyck** [comment]
@sh0rez all changes are ready. Let me know if it's ok. If you think it's ok, we could create another PR with a single commit for readability.

**Pull request merged**